rules/windows/process_creation/proc_creation_win_python_inline_command_execution.yml

```diff
@@ -30,6 +30,8 @@ detection:
         ParentCommandLine|contains: '-E -s -m ensurepip -U --default-pip'
     filter_vscode:
         ParentImage|endswith: '\AppData\Local\Programs\Microsoft VS Code\Code.exe'
+    filter_pip:
+        CommandLine|contains: '<pip-setuptools-caller>'
```

Comment:
Can you link to the log that generated this CLI for historical reasons.

Reply:
This comes from direct compilation of whl packages in the CLI. See https://github.com/search?q=%22This+is+%3Cpip-setuptools-caller%3E%22&type=code